pkg/core/klinebuilder.go

```diff
@@ -0,0 +1,104 @@
+package core
+
+import (
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	log "github.com/sirupsen/logrus"
+)
+
+type KLineBuilder interface {
```

Comment:
but why do we need interface here?

Reply:
Because I want to hide the implementation, the `klineBuilder`, from the outside of this module.